Source/Core/Core/HW/DSP.cpp

```diff
@@ -212,6 +212,7 @@ void EnableInstantDMA()
 	CoreTiming::RemoveEvent(et_CompleteARAM);
 	CompleteARAM(0, 0);
 	instant_dma = true;
+	ERROR_LOG(DSPINTERFACE, "Enabling Instant ARAM DMA hack");
```

Comment:
For whatever reason, Notice is higher than Error, so it might be an option?
+/-0 on whether such messages should exist or not (they might help when looking at a log when it's wrongly active, but might aswell confuse users).